### DISCUSSION TITLE
How to install on Xiaomi Mi 10 / Android 13 phone?

### DISCUSSION THREAD

**Discussion by C-O-D** [rustdesk/rustdesk#8025] [General]
None of the APKs of the last versions work, they are all displayed as invalid during installation. 
Last working version was 1.2.3-2

**Comment by dif1754** [comment]
I've installed it on Redmi Note 13 5Ghz  the  .apk version 1.3.1 for  "AArch64 (ARM64)" architecture . 
It works.
btw, to enable the "input control" on the app (section "Share screen" ), please go first to the Android settings of the app  and enable  "Allow restricted settings", then  try again (it will ask you to enable the accessibility features for the Rustdesk app).
